btree/palm/tree.go

```diff
@@ -35,13 +35,21 @@ const (
 	apply
 )
 
-const multiThreadAt = 1000 // number of keys before we multithread lookups
+const multiThreadAt = 400 // number of keys before we multithread lookups
```

Comment:
Just a general comment, it would be really nice if there was a specific benchmark you could run that would give some hint what the optimal for this was for your system and that it was noted in the readme how to figure it out.